package.json

```diff
@@ -21,5 +23,21 @@
     "react-dom": "18.2.0",
     "tailwindcss": "3.3.2",
     "typescript": "5.1.6"
+  },
+  "devDependencies": {
+    "@jest/globals": "^29.6.1",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^14.0.0",
+    "@testing-library/react-hooks": "^8.0.1",
+    "@testing-library/user-event": "^14.4.3",
+    "@types/jest": "^29.5.2",
+    "@types/testing-library__jest-dom": "^5.14.7",
+    "babel-jest": "^29.6.0",
+    "eslint-plugin-jest-dom": "^5.0.1",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^29.6.0",
+    "jest-environment-jsdom": "^29.6.1",
+    "react-test-renderer": "^18.2.0",
```

Comment:
Please, remove this thing as well. It does needed for testing, you see examples of usage here: https://legacy.reactjs.org/docs/test-renderer.html